test/e2e_node/cpu_manager_test.go

```diff
@@ -557,6 +559,68 @@ func runMultipleCPUContainersGuPod(ctx context.Context, f *framework.Framework)
 	waitForContainerRemoval(ctx, pod.Spec.Containers[1].Name, pod.Name, pod.Namespace)
 }
 
+func runCfsQuotaGuPods(ctx context.Context, f *framework.Framework) {
+	var err error
+	var ctnAttrs []ctnAttribute
+	var pod1, pod2, pod3 *v1.Pod
+	cfsCheckCommand := []string{"sh", "-c", "cat /sys/fs/cgroup/cpu/cpu.cfs_quota_us && sleep 1d"}
```

Comment:
this can be the culprit, we need to check the CI OS supports this path

Reply:
KernelVersion:6.6.56+,OSImage:Container-Optimized OS from Google,ContainerRuntimeVersion:containerd://1.7.23,KubeletVersion:v1.32.0-beta.0.467+cc29a35d38dd82,KubeProxyVersion:v1.32.0-beta.0.467+cc29a35d38dd82,OperatingSystem:linux,Architecture:amd64,}

Reply:
ok, then maybe I'm off the mark. Didn't manage to debug the new tests myself yet.

Reply:
Turns out that I was not off the mark too much: the path is for cgroup v1 tests. We default (and recommend) to v2, so I'd assume cgroup v2 in these tests.
Is not worth to check for cgroup v1 correctness. If the tests cover cgroup v2 that's good enough.

TL;DR: the tests are assuming cgroup v1. They must assume (and clearly state so, possibly with an explicit check) they run on a cgroup v2 machine.